public/js/webcam.js

```diff
@@ -95,17 +94,16 @@ tryAgainButton.addEventListener("click", () => {
 });
 
 function hideAndShowHTMLElementsForTryAgainButton() {
-    video.style.display = "block";
-    takePhotoButton.style.display = "block";
-    tryAgainButton.style.display = "none";
-    getTracksButton.style.display = "none";
-    canvas.style.display = "none";
+    video.className = "video-no-animation";
```

Comment:
I thought it was kind if weird for the animation to play when the user takes the photo again, cause it took a while, but lmk what u think

Reply:
Hmm, that's a good point! The animation is no longer playing for me when I first load it up for some reason, is it working for you?

Reply:
oh what that's weird, did you try refreshing?